volumina/view3d/glview.py

```diff
@@ -190,3 +196,46 @@ def mouseReleaseEvent(self, event):
         """
         self._slice_planes.release()
         return GLViewWidget.mousePressEvent(self, event)
+
+
+class GLViewMock(QLabel):
+    slice_changed = pyqtSignal()
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.setText("3D widget disabled via $HOME/.voluminarc or environment variable\n" "FEATURES_USE_OPENGL_WIDGET.")
+
+    def add_mesh(self, name, mesh=None):
+        pass
+
+    def remove_mesh(self, name):
+        pass
+
+    def is_cached(self, name):
+        return False
+
+    def invalidate_cache(self, name):
+        pass
+
+    @property
+    def visible_meshes(self):
+        return []
+
+    @property
+    def slice(self):
+        return (0, 0, 0)
+
+    @slice.setter
+    def slice(self, slice_):
+        pass
+
+    def set_shape(self, shape):
+        pass
+
+    shape = property(fset=set_shape)
+
+    def toggle_slice(self, axis, visible):
+        pass
+
+
+GLView = volumina.config.CONFIG.use_opengl_viewport and GLViewReal or GLViewMock
```

Comment:
Boolean operators are somewhat confusing here. Maybe change it to `if` expression/statement?